client/my-sites/domains/domain-management/transfer/transfer-out/select-ips-tag.jsx

```diff
@@ -0,0 +1,220 @@
+			this.hideSuggestions();
+		}
+	}
+
+	receiveIpsTagList = ipsTagList => {
+		this.setState( { ipsTagList } );
+	}
+
+	handleKeyDown = event => this.suggestionsRef.handleKeyEvent( event );
+
+	handleSearchInputChange = query => this.setState( { currentQuery: query, ipsTagInput: query } );
+
+	handleSuggestionClick = position => {
+		const parsedLabel = position.label.split( ' ' );
+		this.setState( { ipsTagInput: parsedLabel[ 0 ] } );
+	}
+
+	handleSubmit = () => {
+		this.props.saveDomainIpsTag( this.props.selectedDomainName, this.state.selectedRegistrar );
+	}
+
+	getRegistrarInfo( ipsTag, ipsTagList ) {
+		return find( ipsTagList, [ 'tag', ipsTag ] );
+	}
+
+	getSuggestions() {
+		return this.state.ipsTagList
+			.filter( hint => this.state.currentQuery && startsWith( hint.tag, toUpper( this.state.currentQuery ) ) )
+			.map( hint => ( { label: hint.tag + '  (' + hint.registrarName + ')' } ) );
+	}
+
+	setSuggestionsRef = ref => ( this.suggestionsRef = ref );
+
+	hideSuggestions = () => this.setState( { currentQuery: '' } );
+
+	popOverDialog = () => {
+		const { ipsTagInput, ipsTagList } = this.state;
+		let selectedRegistrar = this.getRegistrarInfo( ipsTagInput, ipsTagList );
+
+		if ( isEmpty( selectedRegistrar ) ) {
+			selectedRegistrar = { tag: toUpper( ipsTagInput ), registrarName: '', registrarUrl: '' };
+		}
+
+		this.setState( {
+			selectedRegistrar,
+			showDialog: true,
+		} );
+	}
+
+	onCloseDialog = ( action ) => {
+		if ( 'submit' === action ) {
+			this.handleSubmit();
+		}
+
+		this.setState( { showDialog: false } );
+	}
+
+	renderGoToGainingRegistrar() {
+		const { translate, gainingRegistrar: { registrarUrl } } = this.props;
+
+		return (
+			<Card
+				className="transfer-out__select-ips-tag"
+				href={ registrarUrl }
+				target="_blank"
+				highlight="success"
+			>
+			{
+				registrarUrl
+				? translate( 'Success! Click here to go to your new registrar and complete the transfer.' )
+				: translate( 'Success! Contact your new registrar to complete the transfer.' )
+			}
+			</Card>
+		);
+	}
+
+	renderIpsTagSelect() {
+		const { selectedDomainName, translate } = this.props;
+
+		return (
+			<div>
+				<p>
+					{ translate(
+						'Please enter the IPS tag of the registrar you wish to transfer ' +
+						'{{strong}}%(selectedDomainName)s{{/strong}} to.',
+						{ args: { selectedDomainName }, components: { strong: <strong /> } }
+					) }
+				</p>
+
+				<SearchCard
+					disableAutocorrect
+					onSearch={ this.handleSearchInputChange }
+					onBlur={ this.hideSuggestions }
+					onKeyDown={ this.handleKeyDown }
+					placeholder="Start typing an IPS tag..."
+					value={ this.state.ipsTagInput }
+				/>
+				<Suggestions
+					ref={ this.setSuggestionsRef }
+					query={ this.state.currentQuery }
+					suggestions={ this.getSuggestions() }
+					suggest={ this.handleSuggestionClick }
+				/>
+				<FormButton onClick={ this.popOverDialog } >
+					{ this.props.translate( 'Submit' ) }
```

Comment:
You're already deconstructing `translate` so we can dispense with `this.props`

Reply:
Woop, yep, artifact from old copy pasta.